elements/howto-tabs/howto-tabs.js

```diff
@@ -57,10 +70,15 @@
       this.attachShadow({mode: 'open'});
       // Import the shared template to create the slots for tabs and panels.
       this.shadowRoot.appendChild(
-        document.importNode(shadowDOMTemplate.content, true)
+        document.importNode(template.content, true)
```

Comment:
I was wondering recently if we have to keep using document.importNode cuz we're not pulling in anything from other documents